mule-extensions-api/src/main/java/org/mule/runtime/extension/api/introspection/property/DisplayModelPropertyBuilder.java

```diff
@@ -48,14 +50,15 @@ private DisplayModelPropertyBuilder()
         this.tabName = "";
     }
 
-    public DisplayModelPropertyBuilder(String displayName, boolean isPassword, boolean isText, int order, String tabName, String groupName)
+    public DisplayModelPropertyBuilder(String displayName, boolean isPassword, boolean isText, int order, String tabName, String groupName, String summary)
```

Comment:
what's the sense for this? A builder with such a constructor is equivalent to just exposing that constructor in the built object. I see that this was already there before your commit, but do take the chance to fix this